src/TriggerBinding/SqlTriggerBinding.cs

```diff
@@ -44,15 +45,17 @@ internal sealed class SqlTriggerBinding<T> : ITriggerBinding
         /// <param name="leasesTableName">Optional - Name of the leases table</param>
         /// <param name="parameter">Trigger binding parameter information</param>
         /// <param name="hostIdProvider">Provider of unique host identifier</param>
+        /// <param name="sqlOptions"></param>
         /// <param name="logger">Facilitates logging of messages</param>
         /// <param name="configuration">Provides configuration values</param>
-        public SqlTriggerBinding(string connectionString, string tableName, string leasesTableName, ParameterInfo parameter, IHostIdProvider hostIdProvider, ILogger logger, IConfiguration configuration)
+        public SqlTriggerBinding(string connectionString, string tableName, string leasesTableName, ParameterInfo parameter, SqlOptions sqlOptions, IHostIdProvider hostIdProvider, ILogger logger, IConfiguration configuration)
```

Comment:
Same comment here and elsewhere regarding `IOptions<SqlOptions>`

Reply:
Updated this to use IOptions  here and other places you commented are not created using DI but use the options from there, so we are good. Thanks!